runtime/default/bootstrap

```diff
@@ -0,0 +1,11 @@
+#!/opt/bin/sh
```

Comment:
```suggestion
#!/usr/bin/env sh
```

?

Reply:
Any reason to prefer one or the other? Also are we certain that the alternative works on Lambda? (compiling, publishing the version, creating a test lambda, deploying and testing takes a good amount of time just for validating this little change)

Reply:
I dont know about lambda. But using `/usr/bin/env sh` Is a way which also supports shells which are installed in non standard paths. Its good practice to use.

Reply:
I understand.

Until we have a reason to choose one, I don't want to compile, publish a layer, create a test lambda, deploy and test that it works for no perceived gain. Let's revisit when we have more information.